include/z64animation.h

```diff
@@ -239,95 +239,95 @@ s16 Animation_GetLength(void* animation);
 s16 Animation_GetLastFrame(void* animation);
 
 /*
- * Animation requests
+ * Animation Task Queue
  */
 
 typedef enum {
-    /* 0 */ ANIMENTRY_LOADFRAME,
-    /* 1 */ ANIMENTRY_COPYALL,
-    /* 2 */ ANIMENTRY_INTERP,
-    /* 3 */ ANIMENTRY_COPYTRUE,
-    /* 4 */ ANIMENTRY_COPYFALSE,
-    /* 5 */ ANIMENTRY_MOVEACTOR
-} AnimationType;
+    /* 0 */ ANIMTASK_LOAD_PLAYER_FRAME,
+    /* 1 */ ANIMTASK_COPY,
+    /* 2 */ ANIMTASK_INTERP,
+    /* 3 */ ANIMTASK_COPY_USING_MAP,
+    /* 4 */ ANIMTASK_COPY_USING_MAP_INVERTED,
+    /* 5 */ ANIMTASK_ACTOR_MOVE
+} AnimTaskType;
 
 typedef struct {
-    /* 0x000 */ DmaRequest req;
-    /* 0x020 */ OSMesgQueue msgQueue;
-    /* 0x038 */ OSMesg msg;
-} AnimEntryLoadFrame; // size = 0x3C
+    /* 0x00 */ DmaRequest req;
+    /* 0x20 */ OSMesgQueue msgQueue;
+    /* 0x38 */ OSMesg msg;
+} AnimTaskLoadPlayerFrame; // size = 0x3C
 
 typedef struct {
-    /* 0x000 */ u8 queueFlag;
-    /* 0x001 */ u8 vecCount;
-    /* 0x004 */ Vec3s* dst;
-    /* 0x008 */ Vec3s* src;
-} AnimEntryCopyAll; // size = 0xC
+    /* 0x00 */ u8 group;
+    /* 0x01 */ u8 vecCount;
+    /* 0x04 */ Vec3s* dest;
+    /* 0x08 */ Vec3s* src;
+} AnimTaskCopy; // size = 0xC
 
 typedef struct {
-    /* 0x000 */ u8 queueFlag;
-    /* 0x001 */ u8 vecCount;
-    /* 0x004 */ Vec3s* base;
-    /* 0x008 */ Vec3s* mod;
-    /* 0x00C */ f32 weight;
-} AnimEntryInterp; // size = 0x10
+    /* 0x00 */ u8 group;
+    /* 0x01 */ u8 vecCount;
+    /* 0x04 */ Vec3s* base;
+    /* 0x08 */ Vec3s* mod;
+    /* 0x0C */ f32 weight;
+} AnimTaskInterp; // size = 0x10
 
 typedef struct {
-    /* 0x000 */ u8 queueFlag;
-    /* 0x001 */ u8 vecCount;
-    /* 0x004 */ Vec3s* dst;
-    /* 0x008 */ Vec3s* src;
-    /* 0x00C */ u8* copyFlag;
-} AnimEntryCopyTrue; // size = 0x10
+    /* 0x00 */ u8 group;
+    /* 0x01 */ u8 vecCount;
+    /* 0x04 */ Vec3s* dest;
+    /* 0x08 */ Vec3s* src;
+    /* 0x0C */ u8* copyMap;
```

Comment:
`limbSet`? This looks to me like an array of booleans indexed by limb denoting which set of limbs to copy

Reply:
That is exactly what it is yeah. I think a "copy map" is a good name for it, it maps out each limb to flag which denotes if it should be copied. I can make it longer with "limbCopyMap" if that helps. 
I already called the array in player a "limb copy map" in a previous round of documentation fwiw. but if we decide it should change then I can change it there too.

I personally dont find that "limb set" tells me much. set what? or do you mean a set of limbs?

Reply:
yeah I think i still like calling it a copy map. does `limbCopyMap` help get my meaning across better?

Reply:
I think I do slightly prefer `limbCopyMap` since it hints that this is an array indexed by limb